src/display_controller.py

```
@@ -343,8 +347,83 @@ def config_change_callback(old_config: Dict[str, Any], new_config: Dict[str, Any
+            logger.debug("Vegas mode disabled in config")
+            return
+
+        try:
+            # Lazy import to avoid circular imports
+            if not _vegas_mode_imported:
+                try:
+                    from src.vegas_mode import VegasModeCoordinator as VMC
+                    VegasModeCoordinator = VMC
+                    _vegas_mode_imported = True
+                except ImportError as ie:
+                    logger.error("Failed to import Vegas mode module: %s", ie)
```

Ruff (0.14.14)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)